### PR TITLE
feature(importer-ergonode): Create ergonode import based on CSV files

### PR DESCRIPTION
**Pull request by BastekBielawski**
# Description

Issue #

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [x] Unit Tests
- [x] Behat Test

# Checklist:
- [x] I have read the contribution requirements and fulfill them.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by BastekBielawski** [comment]
I've update all branch with current master and update all the changes

File: .env [hunk 1]
```diff
@@ -63,3 +63,7 @@ MAILER_DSN=null://null # send email into space :)
 #MAILER_DSN=smtp://localhost:1025 # Mailhog configuration
 MAILER_SENDER=dev@ergonode.com
 ###< symfony/mailer ###
+
+###> behat ###
+BEHAT_MEMORY_LIMIT=1G
```
Comment (BastekBielawski) [inline .env:68]: @rprzedzik Why do u think this solution is a bad idea?

**Review by piotrkreft — COMMENTED** [review]
@BastekBielawski please merge the master branch once again. CI has changed.@BastekBielawski please merge the master branch once again. CI has changed.

**Comment by piotrkreft** [comment]
Closing this as the work will continue from my fork.

**Pull request closed without merging**